src/main/resources/db/migration/V1_70__referral_assignment_history.sql

```diff
@@ -0,0 +1,21 @@
+create table referral_assignments
+(
+    referral_id    uuid not null,
+    assigned_at    timestamp with time zone,
+    assigned_by_id text,
+    assigned_to_id text,
+    constraint fk_referral_id foreign key (referral_id) references referral,
+    constraint fk_assigned_by_id_auth_user foreign key (assigned_by_id) references auth_user,
+    constraint fk_assigned_to_id_auth_user foreign key (assigned_to_id) references auth_user
+);
+
+insert into referral_assignments
+select id, assigned_at, assigned_by_id, assigned_to_id
+from referral;
+
+alter table referral
+    drop column assigned_at;
```

Comment:
_nit_ rename would provide a simpler way to rollback, but I appreciate that's probably not necessary now

Reply:
done